Makefile

```diff
@@ -126,6 +126,7 @@ test-cmd: build
 # Example:
 #   make test-end-to-end
 test-end-to-end: build
+	hack/env hack/verify-generated-protobuf.sh # Test the protobuf serializations when we know Docker is available
```

Comment:
That applies to conversions, clients, and deep copies too.  Agree we should
have something to test, but should be done together.

On Fri, Jul 22, 2016 at 2:09 PM, David Eads <notifications@github.com>
wrote:

> In Makefile
> <https://github.com/openshift/origin/pull/9814#discussion_r71921067>:
>
> > @@ -126,6 +126,7 @@ test-cmd: build
> >  # Example:
> >  #   make test-end-to-end
> >  test-end-to-end: build
> > +	hack/env hack/verify-generated-protobuf.sh # Test the protobuf serializations when we know Docker is available
>
> It does not check upstream protobuf, what do you mean by "additional
> checks"?
>
> We had a couple API carries since shipped a v1 first. We need to make
> sure that never happens again or if it does, make sure that its not
> serialized to protobuf.
>
> I guess we could all promise to do no evil.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9814/files/f605a623322421bf84d1623a3037b41470c530d3..32e60e560f6d18abd70f9903411b694f01c2d539#r71921067>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABG_p9exoCq9ixRWXbfQpjQ600E1jgaVks5qYQdugaJpZM4JLFc7>
> .
>